SignalMetadataKitTests/src/SMKTestUtils.swift

```diff
@@ -253,4 +67,23 @@ class MockClient: NSObject {
                               recipientId: recipient.recipientId,
                               deviceId: recipient.deviceId)
     }
+
+    func generatePreKey() -> PreKeyRecord {
```

Comment:
TIOLI `generateMockPreKey` and `generateMockSignedPreKey` to emphasize that this is mock-only functionality?

Reply:
Done